Audible-Altimeter/src/Runner.cpp

```diff
@@ -0,0 +1,10 @@
+#include <Audible-Altimeter-Embedded/ITiny2350.hpp>
+#include <Audible-Altimeter/Runner.hpp>
+
+namespace altimeter {
+
+Runner::Runner(ITiny2350* tiny) { m_tiny_2350 = tiny; }
+
+void Runner::run() { m_tiny_2350->toggle_leds(false); }
```

Comment:
adda printf?

Reply:
I am not doing it in the `Audible-Altimeter` just yet as I want to investigate the best way to log in here, but I added a few printf statements in the Tiny2350 classes!

Reply:
Sounds good!